package.json

```
@@ -1,16 +1,29 @@
 {
   "name": "e2e-smoke-test",
   "version": "0.0.1",
-  "description": "Smoke tests against external interfaces",
-  "main": "index.js",
+  "description": "E2e and smoke tests",
+  "engines": {
+    "npm": ">=6.14.17",
+    "node": ">=14.18.0"
+  },
   "scripts": {
     "test": "playwright test",
```

Huskylle pitää lisätä myös `prepare`-skripti, että Git-hook otetaan käyttöön npm installin jälkeen.
https://typicode.github.io/husky/#/?id=install

```suggestion
    "prepare": "husky install",
    "test": "playwright test",
```